server/controllers/user.js

```diff
@@ -162,10 +206,8 @@ async function severalGetInfos(req, res) {
  * @param {HttpResponseWrapper} res The http response
  */
 async function getInfos(req, res) {
-  let user = String(req.params.pseudo);
-  if (user === "me") {
-    user = req.body.authUser;
-  }
+  const param = String(req.params.pseudo);
+  const user = param === "me" ? req.body._auth.user : param;
```

Comment:
Style nitpick ```suggestion
  const user = (param === "me") ? req.body._auth.user : param;
```

Reply:
The prettier remove them automatically ...